code/modules/mailorder/modr_entries.dm

```diff
@@ -0,0 +1,365 @@
+		desc = "Can't decide on a cereal? Buy them all and save!"
+		order_items = list(/obj/item/reagent_containers/food/snacks/cereal_box,
+		/obj/item/reagent_containers/food/snacks/cereal_box/honey,
+		/obj/item/reagent_containers/food/snacks/cereal_box/tanhony,
+		/obj/item/reagent_containers/food/snacks/cereal_box/roach)
+		cost = PAY_UNTRAINED
+
+
+//Survival Mart: primarily medical, but not exclusively
+ABSTRACT_TYPE(/datum/mail_order/survmart)
+/datum/mail_order/survmart
+
+	oxytank
+		name = "Emergency Oxygen Tank"
+		desc = "An easy-to-use compact gas tank pre-loaded with pure oxygen."
+		order_items = list(/obj/item/tank/emergency_oxygen)
+		cost = PAY_TRADESMAN
+
+	wintercoat
+		name = "Winterized Overcoat"
+		desc = "Beat the cold of space with your very own cozy coat."
+		order_items = list(/obj/item/clothing/suit/wintercoat)
+		cost = PAY_TRADESMAN / 2
+
+	cleangear
+		name = "Medical Wear Bundle"
+		desc = "Contains latex gloves and sterile mask."
+		order_items = list(/obj/item/clothing/gloves/latex,/obj/item/clothing/mask/surgical)
+		cost = PAY_UNTRAINED / 2
+
+	medbelt
+		name = "Medical Belt"
+		desc = "Top-quality belt for rapid medical equipment access."
+		order_items = list(/obj/item/storage/belt/medical)
+		cost = PAY_DOCTORATE
+		order_perm = list(access_medical)
+
+	analyze_kit
+		name = "Analyzer Triage Bundle"
+		desc = "Order a health analyzer, and we'll throw in a bandage for free!"
+		order_items = list(/obj/item/device/analyzer/healthanalyzer,/obj/item/bandage)
+		cost = PAY_TRADESMAN / 4
+
+	analyze_up
+		name = "Analyzer Deluxe Upgrades"
+		desc = "Top-of-the-line enhancements for any standard health analyzer."
+		order_items = list(/obj/item/device/analyzer/healthanalyzer_upgrade,/obj/item/device/analyzer/healthanalyzer_organ_upgrade)
+		cost = PAY_DOCTORATE / 3
+
+	auto_char
+		name = "Charcoal Auto-Injector"
+		desc = "Rapidly flushes toxins out of your system - a must for biohazards."
+		order_items = list(/obj/item/reagent_containers/emergency_injector/charcoal)
+		cost = PAY_DOCTORATE / 3
+
+	auto_epi
+		name = "Epinephrine Auto-Injector"
+		desc = "Stabilizes critical patients - the gold standard for crisis care."
+		order_items = list(/obj/item/reagent_containers/emergency_injector/epinephrine)
+		cost = PAY_DOCTORATE / 2
+
+	auto_salb
+		name = "Salbutamol Auto-Injector"
+		desc = "Mitigates the effect of hypoxia. Useful in case of hull breach."
+		order_items = list(/obj/item/reagent_containers/emergency_injector/salbutamol)
+		cost = PAY_DOCTORATE / 4
+
+	patch_brute
+		name = "Healing Mini-Patch"
+		desc = "A robust medical patch effective at treating brute-force injury."
+		order_items = list(/obj/item/reagent_containers/patch/mini/bruise)
+		cost = PAY_TRADESMAN / 4
+
+	patch_burn
+		name = "Burn Mini-Patch"
+		desc = "A soothing medical patch effective at treating burn injury."
+		order_items = list(/obj/item/reagent_containers/patch/mini/burn)
+		cost = PAY_TRADESMAN / 4
```

Comment:
Having "on-demand healing" that isn't easily prevented on a map that supports the mail-system method (rather than the QM method) seems like a "meh" idea at best.